function.js

```diff
@@ -97,4 +97,32 @@ closePop.addEventListener('click', (event) => {
     init[i].remove();
   }
   event.preventDefault();
+});
+
+/* ----- Validate Form --- */
+function showMessage(input, message, type) {
+  const msg = input.parentNode.querySelector('small');
+  msg.innerHTML = message;
+  input.className = type ? 'succes' : 'errror';
+  return type;
+}
+
+const valid = document.getElementById('contact-form');
+valid.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const { email } = valid.elements;
+  const s = document.querySelector('.msg');
+
+  if (s.classList.contains('small-on')) s.classList.remove('small-on');
+  if (email.classList.contains('error-email')) email.classList.remove('error-email');
+
+  if (email.value === email.value.toLocaleLowerCase()) {
+    HTMLFormElement.prototype.submit.call(valid);
+    showMessage(email, '', true);
+  } else {
+    console.log('no');
```

Comment:
Great implementation of your validation here. 👍🏿  I would however suggest that you make this tiny fix to your code. 🧰 

- [ ] Please remove `console.log` from your code as it is not a good coding practice in as much as it is a great debugging feature. ♻️